app/grants/models/grant.py

```diff
@@ -24,6 +24,37 @@
 from .subscription import Subscription
 
 
+class GrantPayout(SuperModel):
+    name = models.CharField(
+        max_length=25,
+        help_text=_('Display Name for Payout')
+    )
+    contract_address = models.CharField(
+        max_length=255,
+        null=True,
+        blank=True,
+        help_text=_('Payout Contract from which funds would be claimed')
+    )
+    ready_to_claim = models.BooleanField(default=False, help_text="Is grant payout ready to be claimed?")
+    payout_token = models.CharField(
+        max_length=10,
+        default='DAI',
+        help_text=_('Currency in which funds would be paid')
+    )
+    funding_withdrawn = models.BooleanField(
+        default=False,
+        help_text=_('Was the Matching Contract funding withdrawn?')
+    )
+    funding_withdrawal_date = models.DateTimeField(
+        null=True,
+        blank=True,
+        help_text=_('When was funding in Matching Contract withdrawn?')
+    )
```

Comment:
As opposed to having 

- funding_withdrawn
- ready_to_claim


Why not instead have a single field ? https://github.com/gitcoinco/web/pull/9657/files#r768346087

Reply:
oh yes! i'll change this @thelostone-mc